subprojects/core/src/main/java/org/gradle/internal/fingerprint/CurrentFileCollectionFingerprint.java

```diff
@@ -32,4 +32,6 @@
      * Visits the roots of this file collection fingerprint.
      */
     void visitRoots(PhysicalSnapshotVisitor visitor);
+
+    String getNormalizationStrategyName();
```

Comment:
Why don't we return `FingerprintingStrategy.Identifier` here instead of a plain string?